Sources/PayPalNativeCheckout/PayPalDelegate.swift

```diff
@@ -22,6 +22,16 @@ public protocol PayPalDelegate: AnyObject {
     ///   - client: the PayPalClient associated with delegate
     func paypalDidCancel(_ payPalClient: PayPalClient)
 
+    /// Notify that the PayPal flow is about to start
+    /// - Parameters:
+    ///   - client: the PayPalClient associated with delegate
+    func paypalDidStart(_ payPalClient: PayPalClient)
```

Comment:
The wording here is confusing. `didStart` means the flow has already started, whereas the docstring says "is about to start". Which one is correct?

Reply:
"is about to start" is correct. What wording would you suggest? something like `paypalAboutToStart`?

Reply:
On iOS, I think `will` is pretty common. For example in the `UIViewController` lifecycle [methods](https://developer.apple.com/documentation/uikit/uiviewcontroller), there is `viewWillAppear()` and `viewWillDisappear()`.

We should also clarify in this docstring that an `ASWebAuthenticationSession` will launch, so the merchant can perform any necessary UI customization like spinners or other cleanup.